ui/src/track-layout/track-layout-slice.ts

```diff
@@ -249,14 +249,6 @@ function filterItemSelectOptions(
         }
     }
 
-    if (options?.trackNumbers && options.trackNumbers.length > 0) {
-        options.locationTracks = [];
-    }
-
-    if (options?.locationTracks && options.locationTracks.length > 0) {
-        options.trackNumbers = [];
-    }
-
```

Comment:
Le biff. Otettu nämä pois selection handling -putken loppupäästä, sillä meillä on tilanteita joissa halutaan valita molempia yhtä aikaa

Reply:
Tätäpäs pitääkin vähän miettiä yhteisesti. Raidelinkityksen yhteydessä halutaan valita joko toinen eikä molempia, ja jos toinen on jo valittu niin sen valinta pitäisi nollata ennen toisen valintaa.

Reply:
Joo tuosta tulisi mieleen että meillä pitäisi ehkä olla erilaisia valintatiloja, as in "monivalinta päällä", "single select & vain raiteita/ratanumeroita" jne. Hiukan ärsyttävää tilallisuutta sekin, mutta jos valinnat halutaan pyörittää saman mekanismin kautta niin jotainhan tuohon tarvitaan. 

Toisaalta linkityksessä voisi tietysti olla niin että linkitykseen liittyvät valinnat menee sinne linkitystilaan ja siellä voi olla vain yksi asia, mutta oikeasti valintoja voi olla monta. Tietty siinä ärsyttävä puoli on että pitäisi hanskata valintojen näyttäminen eri tavalla linkityksen ollessa päällä (ehkä silloin ei pitäisi näyttää valinta-layeriä lainkaan vaan vain linkityslayer?)